tests/bucket_policies/test_bucket_policies.py

```diff
@@ -252,3 +254,76 @@ def test_operation_access_policies(
                 f"{op} was {'denied' if not can_access else 'allowed'}"
                 f" for the new account after only allowing {tested_op}"
             )
+
+    @pytest.mark.parametrize(
+        "access_effect",
+        [
+            "Allow",
+            # "Deny", Skipped due to https://bugzilla.redhat.com/show_bug.cgi?id=2280212
+        ],
+    )
```

Comment:
Tracking this with issue https://github.com/red-hat-storage/noobaa-sa-ci/issues/34